Atomic/storage.py

```diff
@@ -19,6 +19,30 @@
 except ImportError:
     from atomic import Atomic
 
+def modify_sh_var_in_text(text, var, modifier, default=""):
+    pattern = '^[ \t]*%s[ \t]*=[ \t]*"(.*)"[ \t]*$' % re.escape(var)
+    found = [ False ]
+    def sub(match):
```

Comment:
Ok, now I know far more about this then I probably wanted.  :^(  Sucks to have to support python2 and python3...